### PR TITLE
feat: add support for checkov

### PR DESCRIPTION
**Pull request by fleroux514**
Adding `setup-checkov` and `checkov-version` Github Action input parameters to setup `checkov`

Example config in digger.yml

```
projects:
  - name: project_a_d
    dir: ./project_a/development
    workflow: project_a

workflows:
  project_a:
    plan:
      steps:
        - init
        - plan
        - run: ./checkov -d . --framework terraform
```

See https://github.com/diggerhq/francios-tests/actions/runs/4999052252/jobs/8955062270

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -126,11 +134,22 @@ runs:
...
+        pip3 install -U checkov==${{ inputs.checkov-version }}
+      shell: bash
+      if: inputs.setup-checkov == 'true'
+
     - name: build and run digger
       if: ${{ !startsWith(github.action_ref, 'v') }}
       shell: bash
       env:
         PLAN_UPLOAD_DESTINATION: ${{ inputs.upload-plan-destination }}
+        SETUP_CHECKOV: ${{ inputs.checkov-version }}
```
Comment (Spartakovic) [inline action.yml]: ${{ inputs.setup-checkov}}

File: pkg/digger/digger.go [hunk 2]
```diff
@@ -418,6 +420,12 @@ func (d DiggerExecutor) Plan(prNumber int) error {
 				},
 			}
 		}
+		if os.Getenv("SETUP_CHECKOV") == "true" && len(planSteps) > 0 {
+			_, _, err := d.CommandRunner.Run(d.RepoPath, planSteps[0].Shell, "source python_venv/bin/activate")
```
Comment (Spartakovic) [inline pkg/digger/digger.go]: I suspect this doesn't do anything, since each command is executed in a new shell

**Comment by Spartakovic** [comment]
> Adding `setup-checkov` and `checkov-version` Github Action input parameters to setup `checkov`
> 
> The binary gets installed under `./python_venv/bin/checkov` and can be executed as this in digger.yml
> 
> ```
> projects:
>   - name: project_a_d
>     dir: ./project_a/development
>     workflow: project_a
> 
> workflows:
>   project_a:
>     plan:
>       steps:
>         - init
>         - plan
>         - run: ../../python_venv/bin/checkov -d . --framework terraform
> ```
> 
> See https://github.com/diggerhq/francios-tests/actions/runs/4999052252/jobs/8955062270

Updated the PR, to avoid having to do "../../python_venv/bin/checkov". It should work with just "checkov"

File: pkg/digger/digger.go (hunk 2 above)
Comment (fleroux514) [reply pkg/digger/digger.go]: Indeed..

**Review by Spartakovic — APPROVED** [review]

**Comment by fleroux514** [comment]
@JamesWoolfenden

It can run by configuring the digger.yaml file, as the example I included in this PR description.

The command would be hardcoded per project

**Pull request merged**